src/library/f3d_window_impl_standard.cxx

```diff
@@ -98,6 +101,12 @@ window_impl_standard::window_impl_standard(const options& options, bool offscree
 {
   this->Internals->RenWin->SetMultiSamples(0); // Disable hardware antialiasing
   this->Internals->RenWin->SetOffScreenRendering(offscreen);
+
+  this->Internals->RenWin->Start();
+
+  // set default window name / icon
+  this->setWindowName(f3d::AppTitle);
```

Comment:
This is incorrect. AppTitle is specific to the application.

Reply:
I'm not sure. We have to provide a default icon/title.

Reply:
I dont think so.